test/rhsmlib/facts/test_cloud_facts.py

```diff
@@ -165,6 +165,7 @@
 AZURE_INSTANCE_ID = "12345678-1234-1234-1234-123456789abc"
 AZURE_SKU = "8.1-ci"
 AZURE_OFFER = "RHEL"
+AZURE_SUBSCRRIPTION_ID = "01234567-0123-0123-0123-012345679abc"
```

Comment:
minor typo: s/SUBSCRRIPTION/SUBSCRIPTION/

Reply:
thanks. pushed the fix.